src/pd/client.rs

```diff
@@ -79,6 +95,10 @@ impl PdClient for RpcClient {
     }
 
     fn bootstrap_cluster(&self, stores: metapb::Store, region: metapb::Region) -> Result<()> {
+        let label = "bootstrap_cluster";
+        let timer = Instant::now();
+        PD_REQUEST_COUNTER_VEC.with_label_values(&[label]).inc();
```

Comment:
The histogram has a counter so we don't need another counter metrics. We can use the counter metrics for region heartbeat especially. 

Btw, I prefer not checking the response header, only recording whole call time is fine.

Reply:
They're different. The counter shows how many requests are initiated, the histogram shows how many requests are responded and their durations.

Reply:
If we don't care the result, the histogram has a counter.